src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -32,11 +32,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            EnsureServices(context.Context);
```

Comment:
Is this gone because this was needed for action selection, which is no longer used?